src/dyncfg/src/dyncfg.proto

```diff
@@ -34,7 +34,12 @@ message ProtoConfigVal {
         bool bool = 2;
         uint32 u32 = 6;
         uint64 usize = 3;
+        ProtoOptionU64 opt_usize = 7;
```

Comment:
Would an `optional uint64` also work or is that not allowed within `oneof`?

Reply:
It's not allowed (the tag is used to declare which oneof variant is present and the `optional uint64` trick requires omitting the tag in the encoded message).